shared-bindings/displayio/Bitmap.c

```diff
@@ -134,7 +134,7 @@ const mp_obj_property_t displayio_bitmap_height_obj = {
 //|
 //|     def __setitem__(self, index: Union[Tuple[int, int], int], value: int) -> None:
 //|         """Sets the value at the given index. The index can either be an x,y tuple or an int equal
-//|         to ``y * width + x``.
+//|         to `y * width + x`.
```

Comment:
Single backquotes here does not make sense, because this is not a cross-reference. Double-backquotes causes the "code" font to be used, so these should stay as double backquotes.